### PR TITLE
Add changelog for 5.34.0

### PR DESCRIPTION
**Pull request by rileykarson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
 ## 5.34.0 (Unreleased)
 
+NOTES:
+* compute: Updated field description of `connection_draining_timeout_sec`, `balancing_mode` and `outlier_detection` in `google_compute_region_backend_service` and `google_compute_backend_service`  to inform that default values will be changed in 6.0.0 ([#18399](https://github.com/hashicorp/terraform-provider-google/pull/18399))
+
+DEPRECATIONS:
+* vertexai: deprecated beta field `embedding_management` for `google_vertex_ai_feature_online_store` resource ([#18348](https://github.com/hashicorp/terraform-provider-google/pull/18348))
```
Comment (SarahFrench) [inline CHANGELOG.md:7]: This field is beta-only, so it might be worth restricting to just the beta changelog?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+* compute: added update support to 'ssl_policy' field in 'compute_region_target_https_proxy' resource ([#18361](https://github.com/hashicorp/terraform-provider-google/pull/18361))
+* compute: removed enum validation on `guest_os_features.type` in `google_compute_disk` to allow for new features to be used without provider update ([#18331](https://github.com/hashicorp/terraform-provider-google/pull/18331))
+* compute: updated documentation of google_compute_target_https_proxy and google_compute_region_target_https_proxy ([#18358](https://github.com/hashicorp/terraform-provider-google/pull/18358))
+* container: added support for `security_posture_config.mode` value "ENTERPRISE" in `resource_container_cluster` ([#18334](https://github.com/hashicorp/terraform-provider-google/pull/18334))
+* discoveryengine: added `document_processing_config` field to `google_discovery_engine_data_store` resource ([#18350](https://github.com/hashicorp/terraform-provider-google/pull/18350))
+* edgecontainer: added 'maintenanceExclusions' field to 'edgecontainer_cluster' resource ([#18370](https://github.com/hashicorp/terraform-provider-google/pull/18370))
```
Comment (SarahFrench) [inline CHANGELOG.md:25]:
```suggestion
* edgecontainer: added 'maintenance_exclusions' field to 'google_edgecontainer_cluster' resource ([#18370](https://github.com/hashicorp/terraform-provider-google/pull/18370))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+* **New Resource:** `google_scc_management_organization_security_health_analytics_custom_module` ([#18374](https://github.com/hashicorp/terraform-provider-google/pull/18374))
+
+IMPROVEMENTS:
+* alloydb: changed the resource `google_alloydb_instance` to be created directly with public IP enabled instead of creating the resource with public IP disabled and then enabling it ([#18344](https://github.com/hashicorp/terraform-provider-google/pull/18344))
+* bigtable: added `automated_backup_configuration` field to `google_bigtable_table` resource ([#18335](https://github.com/hashicorp/terraform-provider-google/pull/18335))
+* cloudbuildv2: added support for connecting to Bitbucket Data Center and Bitbucket Cloud with the `bitbucket_data_center_config` and `bitbucket_cloud_config` fields in `google_cloudbuildv2_connection` ([#18375](https://github.com/hashicorp/terraform-provider-google/pull/18375))
+* compute: added update support to 'ssl_policy' field in 'compute_region_target_https_proxy' resource ([#18361](https://github.com/hashicorp/terraform-provider-google/pull/18361))
```
Comment (SarahFrench) [inline CHANGELOG.md:20]:
```suggestion
* compute: added update support to `ssl_policy` field in `google_compute_region_target_https_proxy` resource ([#18361](https://github.com/hashicorp/terraform-provider-google/pull/18361))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+* container: added support for `security_posture_config.mode` value "ENTERPRISE" in `resource_container_cluster` ([#18334](https://github.com/hashicorp/terraform-provider-google/pull/18334))
+* discoveryengine: added `document_processing_config` field to `google_discovery_engine_data_store` resource ([#18350](https://github.com/hashicorp/terraform-provider-google/pull/18350))
+* edgecontainer: added 'maintenanceExclusions' field to 'edgecontainer_cluster' resource ([#18370](https://github.com/hashicorp/terraform-provider-google/pull/18370))
+* gkehub: added `prevent_drift` field to ConfigManagement `fleet_default_member_config` ([#18330](https://github.com/hashicorp/terraform-provider-google/pull/18330))
+* netapp: added `administrators` field to `google_netapp_active_directory` resource ([#18333](https://github.com/hashicorp/terraform-provider-google/pull/18333))
+* networksecurity: added field update tests for `google_network_securtiy_tls_inspection_policy` ([#18371](https://github.com/hashicorp/terraform-provider-google/pull/18371))
```
Comment (SarahFrench) [inline CHANGELOG.md:28]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,39 @@
...
+* vertexai: promoted `optimized` field to GA for `google_vertex_ai_feature_online_store` resource ([#18348](https://github.com/hashicorp/terraform-provider-google/pull/18348))
+* workbench: updated the metadata keys managed by the backend. ([#18367](https://github.com/hashicorp/terraform-provider-google/pull/18367))
```
Comment (SarahFrench) [inline CHANGELOG.md:30]: Might be worth including the `google_workbench_instance` name, but I'm not sure how it'd be phrased

Comment (hao-nan-li) [reply CHANGELOG.md:26]: Will keep the current description for now to avoid creation confusion on a potential misleading change.

**Pull request merged**